lib/FaceLandmarksExtractor/FaceLandmarksExtractor.py

```diff
@@ -105,25 +105,27 @@ def get_pts_from_predict(a, center, scale):
     c += 0.5
     return [ transform (c[i], center, scale, a.shape[2]) for i in range(a.shape[0]) ]
 
-def initialize(scale_to=2048):
+def initialize(detector, scale_to=2048):
     global dlib_detectors
     global keras_model
     global is_initialized
-    if not is_initialized:       
-        
-        dlib_cnn_face_detector_path = os.path.join(os.path.dirname(__file__), "mmod_human_face_detector.dat")
-        if not os.path.exists(dlib_cnn_face_detector_path):
+    if not is_initialized:
+        try:
+            dlib_cnn_face_detector_path = os.path.join(os.path.dirname(__file__), "mmod_human_face_detector.dat")
+        except:
```

Comment:
wtf ?

Reply:
I think what he means here is that a try/except without actually handling the error is an inappropriate solution to error handling.  The exception should either be dealt with (or if ignored, at least an error printed to the console) or passed up to the caller to handle.